packages/cspell-eslint-plugin/src/synckit/index.cts

```diff
@@ -0,0 +1,13 @@
+import { createSyncFn as _createSyncFn } from 'synckit';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+type AnyFn = (...args: any[]) => any;
+type Syncify<T extends AnyFn> = (...args: Parameters<T>) => Awaited<ReturnType<T>>;
+
+export type CreateSyncFn<T extends AnyFn> = (workerPath: URL | string, timeoutOrOptions?: number) => Syncify<T>;
+
+export function createSyncFn<T extends AnyFn>(workerPath: URL | string, timeoutOrOptions?: number): Syncify<T> {
```

Comment:
What's the upstream typing issue here?

Reply:
@JounQin 

Here is the error:

<img width="1063" alt="image" src="https://github.com/user-attachments/assets/697491b1-1ae1-4b47-8b2e-a834cce3f444" />

`node_modules/synckit/index.d.cts`
```ts
import * as synckit from './lib/index.js'

export = synckit
```

Reply:
Thanks for the screenshot, I'll try to fix it.